roles/pulp_api/tasks/main.yml

```diff
@@ -14,6 +14,17 @@
         # We can also try using seport's ignore_selinux_state variable
         - ansible_facts.selinux.status == "enabled"
 
+    - name: Install pulpcore-api socket file (when it applies)
+      template:
+        src: pulpcore-api.socket.j2
+        dest: /lib/systemd/system/pulpcore-api.socket
+        owner: root
+        group: root
+        mode: 0644
+      become: true
+      notify: Restart pulpcore-api.service
```

Comment:
Note you also need to restart a .socket file if it changes. You may also need to do a daemon-reload (not sure if that's already there).